src/components/29-timeline/timeline--centered-dates.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="rvt-timeline rvt-timeline--center">
+  <div>
+    <div class="rvt-timeline__item">
+      <div class="rvt-timeline__marker" aria-hidden>
```

Comment:
`aria-hidden` needs to have a value set of `true`. Even thought the absence of the the `aria-hidden` attribute is the same as `araia-hidden="false`, `aria-hidden` with no value is not the equivalent of `aria-hidden="true"` (like say `disabled` or `hidden`) as far as browser a11y APIs are concerned. This article has a pretty detailed explanation of the differences.

https://developer.paciellogroup.com/blog/2012/05/html5-accessibility-chops-hidden-and-aria-hidden/

This goes for all instances of `aria-hidden` in the PR